### PR TITLE
Fix detect property value type on import

### LINKED ISSUE #26: Value type detection is broken when importing from CSV/XLSX
Numbers are always detected as strings.

### PR DESCRIPTION
**Pull request by stel**
fixes #26 

/cc @TimOliver, @jpsim

### CODE REVIEW

File: RealmConverter/Schema/ImportSchemaGenerator.swift [hunk 1]
```diff
@@ -117,55 +117,55 @@ public class ImportSchemaGenerator: NSObject {
...
+                        }
+
+                        guard property.type == .String else {
+                            return
+                        }
+
+                        if let number = numberFormatter.numberFromString(value) {
+                            let numberType = CFNumberGetType(number)
+                            switch (numberType) {
+                            case .SInt8Type: fallthrough
```
Comment (jpsim) [inline RealmConverter/Schema/ImportSchemaGenerator.swift]: you can just comma-separate cases, which will be more readable than `fallthrough`.

File: RealmConverterTests/Swift/RealmConverter+Importer.swift [hunk 2]
```diff
@@ -101,4 +105,26 @@ class RealmConverter_Importer: XCTestCase {
 
         XCTAssertTrue(NSFileManager.defaultManager().fileExistsAtPath(String(destinationRealmPath)))
     }
+
+    func testThatPropertyTypesAreDetectedProperlyWhenImportingFromCSV() {
+        let csvSchema = try! generateSchemaForFileAtPath(bundle.pathForResource("dogs", ofType: "csv")!)
+        XCTAssertTrue(csvSchema.schemas[0].properties[1].type == .Int)
+    }
+
+    // FIXME: XLSX import doesn't seem to work at all :(
```
Comment (jpsim) [inline RealmConverterTests/Swift/RealmConverter+Importer.swift]: can you please file an issue for this?

**Comment by PanMluvci** [comment]
When this fix will be in Realm Browser?

File: RealmConverterTests/Swift/RealmConverter+Importer.swift (hunk 2 above)
Comment (stel) [inline RealmConverterTests/Swift/RealmConverter+Importer.swift]: sure, done https://github.com/realm/realm-cocoa-converter/issues/29

**Comment by stel** [comment]
@PanMluvci this fix will be included in the next Browser's release as soon as this PR is approved and new version of RealmConverter is released :)

**Comment by jpsim** [comment]
Seems you've _nearly_ set up Travis CI, just missing a `.travis.yml` file

**Comment by stel** [comment]
Merged `master` but something is still wrong with running tests on Travis, so will try to fix before merging.

**Comment by PanMluvci** [comment]
well i added 
` pod 'RealmConverter', :git => 'https://github.com/realm/realm-cocoa-converter', :commit => '9acbe9e'` 
to the realm browser but now it it converting all rows to INT
snimek obrazovky 2016-10-13 v 10 07 44

here is my CSV source

[CarDB.zip](https://github.com/realm/realm-cocoa-converter/files/526510/CarDB.zip)

**Comment by stel** [comment]
@PanMluvci sorry for that and thanks for reporting this issue! I hope I've finally fixed it in #34, could you please try `pod 'RealmConverter', :git => 'https://github.com/realm/realm-cocoa-converter', :branch => 'do-fix-import-value-type-fix'`?

**Pull request merged**